roles/ceph-common/defaults/main.yml

```diff
@@ -7,6 +7,20 @@
 
 fetch_directory: fetch/
 
+###############
+# PERMISSIONS #
+###############
+
+# Permissions for /etc/ceph configuration directory
+conf_directory_owner: root
+conf_directory_group: root
+conf_directory_mode: 644

```

Comment:
In order to be traversed / listed they need to be executable.

https://unix.stackexchange.com/a/21252